opentelekomcloud/provider_test.go

```diff
@@ -30,6 +30,10 @@ var (
 	OS_VPC_ID                 = os.Getenv("OS_VPC_ID")
 	OS_SUBNET_ID              = os.Getenv("OS_SUBNET_ID")
 	OS_TENANT_ID              = os.Getenv("OS_TENANT_ID")
+	OS_KEYPAIR_NAME           = os.Getenv("OS_KEYPAIR_NAME")
+	OS_BMS_FLAVOR_NAME        = os.Getenv("OS_BMS_FLAVOR_NAME")
+	OS_SERVER_ID              = os.Getenv("OS_SERVER_ID")
```

Comment:
Instead of adding a new OS_SERVER_ID, can you please creating a new bms server with the test case?